### ISSUE TITLE
Enum validator retrun wrong error (4.0.3)

### ISSUE THREAD

**Issue by julien-maurel** [Automattic/mongoose#3009]
Hi,
In last version, enum validaotr return a wrong object. "kind" property of the error is not set.
Example : 
```
           "myField": {
              "properties": {
                "kind": "enum",
                "message": "`{VALUE}` is not a valid enum value for path `{PATH}`.",
                "path": "myField",
                "value": "badvalue"
              },
              "message": "`badvalue` is not a valid enum value for path `myField`.",
              "name": "ValidatorError",
              "path": "myField",
              "value": "badvalue"
              (here, other validators have a "kind" property, "enum" has not (in 4.0.2, it was ok))
            }
```
Thanks

**Comment by julien-maurel** [comment]
And normally, properties.kind is properties.type

**Comment by vkarpov15** [comment]
Thanks for pointing this out, will fix.

Re: https://github.com/Automattic/mongoose/issues/3009#issuecomment-104648709, we changed type -> kind in 4.0, see https://github.com/Automattic/mongoose/wiki/4.0-Release-Notes and #1323. Admittedly this was not in the 4.0 release notes originally, I missed it because that issue was one of the first fixed in the 4.0 release cycle (before I joined the project) and thus wasn't labeled correctly. It was added to the wiki release notes in #2886.
